lsp-mode.el

```diff
@@ -479,46 +479,64 @@ the server has requested that."
   :package-version '(lsp-mode . "6.1"))
 ;;;###autoload(put 'lsp-enable-file-watchers 'safe-local-variable #'booleanp)
 
-(defcustom lsp-file-watch-ignored '(; SCM tools
-                                    "[/\\\\]\\.git\\'"
-                                    "[/\\\\]\\.hg\\'"
-                                    "[/\\\\]\\.bzr\\'"
-                                    "[/\\\\]_darcs\\'"
-                                    "[/\\\\]\\.svn\\'"
-                                    "[/\\\\]_FOSSIL_\\'"
-                                    ;; IDE or build tools
-                                    "[/\\\\]\\.idea\\'"
-                                    "[/\\\\]\\.ensime_cache\\'"
-                                    "[/\\\\]\\.eunit\\'"
-                                    "[/\\\\]node_modules"
-                                    "[/\\\\]\\.fslckout\\'"
-                                    "[/\\\\]\\.tox\\'"
-                                    "[/\\\\]dist\\'"
-                                    "[/\\\\]dist-newstyle\\'"
-                                    "[/\\\\]\\.stack-work\\'"
-                                    "[/\\\\]\\.bloop\\'"
-                                    "[/\\\\]\\.metals\\'"
-                                    "[/\\\\]target\\'"
-                                    "[/\\\\]\\.ccls-cache\\'"
-                                    "[/\\\\]\\.vscode\\'"
-                                    ;; Autotools output
-                                    "[/\\\\]\\.deps\\'"
-                                    "[/\\\\]build-aux\\'"
-                                    "[/\\\\]autom4te.cache\\'"
-                                    "[/\\\\]\\.reference\\'"
-                                    ;; .Net Core build-output
-                                    "[/\\\\]bin/Debug\\'"
-                                    "[/\\\\]obj\\'")
+(define-obsolete-variable-alias 'lsp-file-watch-ignored 'lsp-file-watch-ignored-directories "7.1.0")
+
+(defcustom lsp-file-watch-ignored-directories
```

Comment:
Can we somewhere explicitly state what the difference between both variables is? E. g. the files one will filter out the events and it wont result in fewer watches in the OS. I am sure that this will be a source of confusion.